backend/src/backend/constants.py

```diff
@@ -9,14 +9,18 @@
 
 COLLECTION_EXPIRE_AFTER = datetime.timedelta(days=7)
 
+if not os.getenv("POSTGRES_URI"):
+    raise EnvironmentError("Please set the POSTGRES_URI environment variable")
+else:
+    POSTGRES_URI = os.getenv("POSTGRES_URI")
```

Comment:
Ah maybe you misunderstood what I meant. `BackendConf.postgres_uri` is fine. It's just that it was named `database_url` that was problematic.

Reply:
Ok, I will move this part back to `BackendConf`.